src/java/com/rapleaf/jack/BaseDatabaseConnection.java

```diff
@@ -8,20 +8,34 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 
+import com.google.common.base.Optional;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public abstract class BaseDatabaseConnection implements Serializable, Closeable {
 
   private static Logger LOG = LoggerFactory.getLogger(BaseDatabaseConnection.class);
 
+  public static String DISABLE_PROPERTY = "jack.db.disallow_connections";
+  public static String DISABLE_ENV_VAR = "jack.db.disallow_connections";
```

Comment:
it feels like JACK_DB_DISALLOW_CONNECTIONS would be more conventional for the environment variable.